src/EthereumNamingService.ts

```diff
@@ -27,11 +27,11 @@ export abstract class EthereumNamingService extends NamingService {
   };
 
   static readonly UrlMap: BlockhanNetworkUrlMap = {
-    mainnet: 'https://mainnet.infura.io',
-    ropsten: 'https://ropsten.infura.io',
-    kovan: 'https://kovan.infura.io',
-    rinkeby: 'https://rinkeby.infura.io',
-    goerli: 'https://goerli.infura.io',
+    mainnet: 'https://main-rpc.linkpool.io',
+    ropsten: 'https://ropsten-rpc.linkpool.io',
+    rinkeby: '', 
+    goerli: '', 
+    kovan: '', 
```

Comment:
Why would you need a blank string here? 

It is a common convention to prefer undefined instead of empty string in JS.